src/desktop/apps/auction/routes.js

```diff
@@ -73,6 +75,15 @@ export async function index(req, res, next) {
       }
     }
 
+    const registeredToBid =
```

Comment:
Can we call `qualifiedForBidding`? @yuki24 pointed out it's a much better name here and as such [I updated the Reaction side](https://github.com/artsy/reaction/pull/3345/commits/f245d2a30594e82c8d1b1a136b09cc01a205547e).

Reply:
If this is a [somewhat] complex conditional that is also shared amongst clients, perhaps this field could move to MP? As in, the logic to fetch bidders, inspect a sale, etc. can possibly exist as:

```
me {
  qualifiedForBidding(sale_id: Int)
}
```

or, perhaps it makes sense to nest under a sale:

```
sale {
  qualifiedForBidding // may eventually return true for some logged out users, etc.
}
```

Reply:
Our plan is to move the "needing identity verification check" into MP (https://artsyproduct.atlassian.net/browse/AUCT-983) – realized a bit into the Force and [Reaction](https://github.com/artsy/reaction/pull/3345) work that we want this, but didn't want to block progress in the meanwhile.

`sale { registrationStatus { qualifiedForBidding }  }` is already [exposed and used in Reaction](https://github.com/artsy/reaction/pull/3345/files#diff-a89333a7d6531d1630eb154a9fb57fa6R290-R292), so good point here. @bhoggard Could be worth considering if a refactor to the sale-originated query is straightforward.

Reply:
We already use code like

```javascript
const qualifiedForBidding = get(me, "bidders.0.qualified_for_bidding")
```
elsewhere in this component, so I'm not going to refactor that now.